samples/snippets/create_data_labeling_job_image_segmentation_sample.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_create_data_labeling_job_image_segmentation_sample]
+from google.cloud import aiplatform
+from google.protobuf import json_format
+from google.protobuf.struct_pb2 import Value
+
+
+def create_data_labeling_job_image_segmentation_sample(
+    project: str,
+    display_name: str,
+    dataset: str,
+    instruction_uri: str,
+    inputs_schema_uri: str,
+    annotation_spec: dict,
```

Comment:
should this be a `dict` or a `str`?

Reply:
An example usage of annotation_spec for image segmentation labeling is {"color": {"red": 1.0}, "displayName": "rose"}, thus dict is specified instead of str as in i.e. classification.